tools/preprocessing.m

```diff
@@ -211,6 +221,7 @@ function set_defaults(obj)
             %% &WALLS
             preprocessing.addvar(obj, 'iwallmom', 3)
             preprocessing.addvar(obj, 'iwalltemp', 1)
+            preprocessing.addvar(obj, 'lbottom', 0)
```

Comment:
Don't think `lbottom` is actually needed in preprocessing, but fine to have it anyway.

Reply:
Actually I was using it for some tree preprocessing. But later removed that part. Let it be as it is as I might again work on the tree preprocessing. This will be mainly be needed for plotting purpose. I am thinking of having a section at the end of write_inputs for all the check plots. Will see later.